assets/sass/components/user-input/googlesitekit-user-input-module.scss

```diff
@@ -148,3 +186,32 @@
 		}
 	}
 }
+
+// Adapt the fixed position bar based on the folded state of the WordPress menu
+.folded .googlesitekit-plugin .googlesitekit-user-input__footer {
+	left: 36px !important;
```

Comment:
Can this be overridden with stronger selectors instead of using the `important`? Usually `important` is used as a last resort to override inline js style for example since there is no other way around it. If this is this only solution it is useful to leave a comment to clarify the reasoning why `important` had to be used